src/appConfig.js

```diff
@@ -374,7 +375,7 @@ configuration.regions = [
                     "queryProperties": { "Regulation Points": { "Descript": "Description" } }
                 }
             },
-        "Applications": ["Regulation"],
+        "Applications": ["Regulation", "EquationWeighting"],
```

Comment:
Should we rename to something more specific, like ChannelWidthWeighting? Not sure if other types of equation weighting may come up in the future.